src/components/Calculator.js

```diff
@@ -18,39 +18,42 @@ const Calculator = () => {
   };
 
   return (
-    <div className="calculator-container">
-      <div className="display" id="display">
-        <span className="hint">
-          {totals.total}
-          {totals.operation}
-          {totals.next}
-        </span>
-        <span className="total">
-          {totals.next ?? totals.total ?? 0}
-        </span>
-      </div>
-      <div className="button-grid">
-        <CalcButton myFunc={handleClick} buttonName="AC" buttonClasses="btn clear" id="clear" />
-        <CalcButton myFunc={handleClick} buttonName="+/-" buttonClasses="btn plusMinus" id="plus-minus" />
-        <CalcButton myFunc={handleClick} buttonName="%" buttonClasses="btn modulo" id="modulo" />
-        <CalcButton myFunc={handleClick} buttonName="÷" buttonClasses="btn divide orange" id="divide" />
-        <CalcButton myFunc={handleClick} buttonName="7" buttonClasses="btn seven" id="seven" />
-        <CalcButton myFunc={handleClick} buttonName="8" buttonClasses="btn eight" id="eight" />
-        <CalcButton myFunc={handleClick} buttonName="9" buttonClasses="btn nine" id="nine" />
-        <CalcButton myFunc={handleClick} buttonName="x" buttonClasses="btn multiply orange" id="multiply" />
-        <CalcButton myFunc={handleClick} buttonName="4" buttonClasses="btn four" id="four" />
-        <CalcButton myFunc={handleClick} buttonName="5" buttonClasses="btn five" id="five" />
-        <CalcButton myFunc={handleClick} buttonName="6" buttonClasses="btn six" id="six" />
```

Comment:
Please ensure that the general layout matches these wireframes.

Your Implementation | Design
:---:|:---:
![image](https://user-images.githubusercontent.com/10332499/166108172-0335fe26-7727-45d9-85ab-f92cbc69a899.png) | ![image](https://user-images.githubusercontent.com/10332499/166108154-d795f179-9ab6-4e79-a21d-8f52e71ba751.png)

Reply:
I work on a wide screen and it looks good on my end. Changed to a 1080p 16:9 ratio and it still looks good.

Reply:
![snipcalc](https://user-images.githubusercontent.com/90456900/166109703-72bae177-d879-4a87-b2fb-2fce85d887aa.png)

As you can see, it looks fine for me.